_posts/2023-02-06-release.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Release 2023.06
+author: Priya
+date: 2023-02-06
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### New Data addition modal
+Migrated existing APIs & Databases tab to Vue for improved UI for adding data through File, Webhook, URL, and Third-Party connectors.
+
+### Defect fixes
+
+* Fixed Sharepoint dataset creation and loader display issue.
+* Renaming of view was showing a red border and a message in red color for the existing view name.
```

Comment:
Fixed an issue where renaming a View would show error motifs for a legitimate case.